lib/hawk/http.rb

```diff
@@ -107,7 +107,9 @@ def request(method, path, options)
 
     private
       def build_url path
-        base.merge(path.sub(/^\//, '')).to_s
+        uri = base.merge(path.sub(/^\//, ''))
+        uri.path.squeeze!
+        uri.to_s
```

Comment:
this is stripping out double slashes?

Can we have a spec covering the issue this is fixing?

Reply:
Yes, this aims to fix https://github.com/ifad/reticulum-client/issues/18 (and probably some others). I will do it properly (and add tests) later, for now I just needed it to debug people.